docs/demos/jspsych-sketchpad-demo1.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <script src="https://unpkg.com/jspsych@7.0.0"></script>
+    <script src="https://unpkg.com/@jspsych/plugin-html-button-response@1.0.0"></script>
+    <script src="https://unpkg.com/@jspsych/plugin-sketchpad@1.0.0"></script>
+    <link
+      rel="stylesheet"
+      href="https://unpkg.com/jspsych@7.0.0/css/jspsych.css"
+    />
+    <style>
+      .jspsych-btn {
```

Comment:
> it would apparently add a scrollbar in some experiments

That's actually very interesting, so thanks for that example. That seems like something that could be fixed by well-crafted CSS (see [this stackoverflow post](https://stackoverflow.com/questions/16670931/hide-scroll-bar-but-while-still-being-able-to-scroll)) and well-defined problems and use cases. That said I think we've gotten beyond the scope of this review, discussing *future* changes to the main CSS. At that point there may be enough changes that it can just be included in an already planned version bump.